explainaboard/tasks.py

```diff
@@ -157,6 +157,16 @@ class TaskCategory:
             )
         ],
     ),
+    TaskCategory(
+        "text-regression",
+        "evaluate evaluation metrics.  ",
+        [
+            Task(
+                name=TaskType.text_regression,
```

Comment:
we could re-consider the task name, some other options, such as `meta-evaluation`